core/arch/riscv64/riscv64.asm

```diff
@@ -395,11 +394,9 @@ GLOBAL_LABEL(dr_setjmp:)
         fsd      fs10, 24*ARG_SZ (ARG1)
         fsd      fs11, 25*ARG_SZ (ARG1)
 # ifdef UNIX
-        addi     sp, sp, -16
-        sd       ra, 0 (sp)
+        mv       t0, ra
```

Comment:
Based on the RISC-V calling convention, t0 is volatile across calls.

Reply:
Sorry, I reverted this change.